content/blog/eng/v-protetto12-7-2024.md

```diff
@@ -0,0 +1,90 @@
+
+<a href= "/images/venerd%C3%AC_protetto/hackathon_2.png?raw=true" target="_blank">
+<img align="center" style="width:75%; margin: auto" src=/images/venerd%C3%AC_protetto/hackathon_2.png?raw=true" alt="hackathon working on the solution" title="Hackathon working on the solution" /> 
+</a>
+
+*Navigating through React Admin documentation*
+
+## Waste time no more
+
+Last July, our company hosted a hackathon to address one of our time's pressing issues: *waste management*. 
+
+The challenge was to develop an app that would enable citizens to report litter in their surroundings, encouraging individuals to take an active role in keeping our environment clean. The project aimed not only to provide a tool for reporting trash but also to raise awareness about the impact of waste on our ecosystems. 
+
+The hackathon unfolded over an *intense four-hour period*, during which participants collaborated across departments, sharing insights and expertise as they brainstormed features for the app. Because the schedule was very tight, they sought a solution that would yield results in the shortest possible time. 
+
+<div>
+<a href= "/images/venerd%C3%AC_protetto/hackathon_1.png?raw=true" target="_blank"> 
+<img align="center" style="width:75%; margin: auto" src=/images/venerd%C3%AC_protetto/hackathon_1.png?raw=true" alt="hackathon code image" title="Hackathon code image" /> 
+</a>
+
+*Coding the way through the solution*
+
+
+The goal was to use the right technologies to create an app that requires geolocation, database access, and smartphone integration, integrate the app with [OpenStreetMap](https://www.openstreetmap.org/), and utilize [React Admin](https://marmelab.com/react-admin/) to streamline the development process of the webpage, letting us to have different experieces as a user or as ad admin. By the end of the event, a *functioning prototype* equipped with an administrator's panel emerged, showcasing technical capabilities and a deep understanding of the environmental issues at play. We're using these libraries mainly because they're free to use, but also because they're easy to use, with good documentation and good community support.
+
+We also use [Firebase](https://firebase.google.com/) as a temporary database, useful also for mobile version. Firebase is a no-brain-go for hackathon users, because it provides a ready-to-go database, configurable via JSON and with all the basic security settings already on the go.
+
+## Wrapping up
+
+The challenge was undeniably a success, and attendees found the collaborative effort invigorating, as cross-functional teamwork often breeds innovative ideas that wouldn’t have surfaced in a traditional work environment. In conclusion, our hackathon yielded the exciting groundwork for a potentially transformative app and fortified our company culture. 
```

Comment:
If we want to mention T-shirts, here's a suggestion:
The challenge was undeniably a success, and attendees found the collaborative effort invigorating, as cross-functional teamwork often breeds innovative ideas that wouldn’t have surfaced in a traditional work environment. To top it all off, participants who put in great effort for the challenge received a reward: a Facile.it T-shirt.

In conclusion, our hackathon yielded the exciting groundwork for a potentially transformative app and fortified our company culture.